addons/im_livechat/static/tests/embed/livechat_session.test.js

```diff
@@ -112,6 +112,7 @@ test.tags("focus required")("Seen message is saved on the server", async () => {
         ["guest_id", "=", guestId],
         ["channel_id", "=", getService("im_livechat.livechat").thread.id],
     ]);
+    await waitNotifications([env, "mail.record/insert"]);
```

Comment:
Note: state was previously changed locally, now having to wait on server result.